srcpkgs/drawterm-9front/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'drawterm-9front'
+pkgname=drawterm-9front
+version=r393.cdbacd80f61a
```

Comment:
The policy is usually to use a tagged release instead of an arbitrary commit hash.

Reply:
I understand. but what I can do that. as you can [see](https://code.9front.org/hg/drawterm/) they don't provide tags of releases. what do you recommend me?

Reply:
Well that complicates matters. Will have to wait for someone else to weigh in on it. Do you know if they release new binaries after each new commit? If they don't, you should pick the hash used to build the latest binary.

Reply:
There's plans for the future to delivery releases on sourcehut. So  for now I think I'll have to pick the latest hash as you said.
my question is: what format should it have?
the hash: ```version=cdbacd80f61a```
the revision: ```version=r939```

Reply:
definitely revision. Something like

```
version=393

...

hg clone -r r${version} ...
```

Reply:
Perfect! thank you.